test/unit/search/fm_index_cursor/bi_fm_index_cursor_collection_test_template.hpp

```diff
@@ -222,5 +222,38 @@ TYPED_TEST_P(bi_fm_index_cursor_collection_test, to_rev_cursor)
     }
 }
 
+TYPED_TEST_P(bi_fm_index_cursor_collection_test, locate_char_string)
```

Comment:
```suggestion
TYPED_TEST_P(bi_fm_index_cursor_collection_test, extend_const_char_pointer)
```
:nail_care: Since it is not about locate but about extend_left/right :)